test_runner/regress/test_tenants.py

```diff
@@ -161,6 +162,15 @@ def test_metrics_normal_work(neon_env_builder: NeonEnvBuilder):
             f"process_start_time_seconds (UTC): {datetime.fromtimestamp(metrics.query_one('process_start_time_seconds').value)}"
         )
 
+    # Test (a subset of) pageserver global metrics
+    for metric in PAGESERVER_GLOBAL_METRICS:
+        ps_samples = ps_metrics.query_all(metric, {})
+        assert len(ps_samples) > 0
+        for sample in ps_samples:
+            labels_str = ",".join([f'{key}="{value}"' for key, value in sample.labels.items()])
+            line = "{0}{{{1}}} {2}".format(sample.name, labels_str, sample.value)
+            log.info(line)
```

Comment:
What kind of testing idea was there, this only logs?

Reply:
This does log these three components (histo + sum + count) with only ["operation"], no tenant or timeline. This seems wrong or does prometheus summarize values with more labels them to the one label item?

Reply:
Oki now I understand, histogram has only the operation, count and sum have tenant and timeline. I'll try to add this as a comment.